submitit/helpers.py

```diff
@@ -293,10 +293,16 @@ def monitor_jobs(
 
 
 @contextlib.contextmanager
-def clean_env() -> tp.Iterator[None]:
+def clean_env(extra_names: tp.Optional[tp.Sequence[str]] = None) -> tp.Iterator[None]:
```

Comment:
as a tuple is a sequence and is not mutable, you can initialize with an empty tuple and avoid dealing with the None case
```suggestion
def clean_env(extra_names: tp.Optional[tp.Sequence[str]] = ()) -> tp.Iterator[None]:
```

Reply:
yes, it makes sense, just force of habit